intl/msg_hash_us.h

```diff
@@ -8283,6 +8507,10 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_SHADER_APPLY_CHANGES,
    "Changes to the shader configuration will take effect immediately. Use this if you changed the amount of shader passes, filtering, FBO scale, etc."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_SHADER_APPLY_CHANGES,
+   "Apply Shader Changes. After changing shader settings such as amount of shader passes, filtering, FBO scale, use this to apply changes.\nChanging these shader settings is a somewhat expensive operation so it has to be done explicitly.\nWhen you apply shaders, the menu shader settings are saved to a temporary file (either menu.cgp or menu.glslp) and loaded. The file persists after RetroArch exits. The file is saved to Shader Directory."
```

Comment:
Possible bad inheritance here: that first line seems to have no reason of being there.

Reply:
Noted, same comment as previous.